### PR TITLE
Remove UAA support

### PR DESCRIPTION
**Pull request by pascalgrimaud**
Fix https://github.com/jhipster/generator-jhipster/issues/13081

### CODE REVIEW

File: generators/common/templates/README.md.ejs [hunk 1]
```diff
@@ -437,8 +435,7 @@ To configure CI for your project, run the ci-cd sub-generator (`jhipster ci-cd`)
 
 [JHipster Homepage and latest documentation]: <%= DOCUMENTATION_URL %>
 [JHipster <%= jhipsterVersion %> archive]: <%= DOCUMENTATION_ARCHIVE_URL %>
-<% if (applicationType === 'gateway' || applicationType === 'microservice' || applicationType === 'uaa') { %>[Doing microservices with JHipster]: <%= DOCUMENTATION_ARCHIVE_URL %>/microservices-architecture/<% } %>
-<%_ if (applicationType === 'uaa') { _%>[Using UAA for Microservice Security]: <%= DOCUMENTATION_ARCHIVE_URL %>/using-uaa/<%_ } _%>
+<% if (applicationType === 'gateway' || applicationType === 'microservice') { %>[Doing microservices with JHipster]: <%= DOCUMENTATION_ARCHIVE_URL %>/microservices-architecture/<% } %>
```
Comment (mshima) [inline generators/common/templates/README.md.ejs:438]:
```suggestion
<%_ if (applicationType === 'gateway' || applicationType === 'microservice') { _%>[Doing microservices with JHipster]: <%= DOCUMENTATION_ARCHIVE_URL %>/microservices-architecture/<%_ } _%>
```

**Review by mshima — APPROVED** [review]
LGTM.
Just a missing condition removal and an empty line removal.

**Comment by pascalgrimaud** [comment]
Thanks so much for the review @mshima !
I'll update this PR soon

**Comment by mshima** [comment]
Great work @pascalgrimaud

**Comment by pascalgrimaud** [comment]
As approved by @mshima, I'm merging this, to avoid too much conflicts in incoming PR

**Pull request merged**